packages/loader/container-loader/src/loader.ts

```diff
@@ -437,7 +432,7 @@ export class Loader implements IHostLoader {
                 canReconnect: request.headers?.[LoaderHeader.reconnect],
                 clientDetailsOverride: request.headers?.[LoaderHeader.clientDetails],
                 resolvedUrl: resolved,
-                version: request.headers?.[LoaderHeader.version],
+                version: request.headers?.[LoaderHeader.version] ?? undefined,
```

Comment:
same here

Reply:
request.headers?.[LoaderHeader.version] returns null if not present. So needs to return undefined.